.github/workflows/vcpkg_ci_linux.yml

```diff
@@ -122,8 +144,13 @@ jobs:
         shell: 'bash'
         # NOTE: Cannot use --clean-after-build because of path issue finding semantics files
         run: |
+          # Doesn't work on arm64
```

Comment:
Is this comment still relevant? Seems a bit off since we say something doesnt work on arm64 and proceed to do an `if` on arm64.